lib/docs/src/tag.stories.tsx

```diff
@@ -1,61 +1,56 @@
 import { Meta } from "@storybook/react";
-import { Fragment } from "react";
+import { Fragment, useState } from "react";
 import { Button, DivPx, Tag } from "../../core/src";
 import { Utils } from "./utils";
-
-const colors: string[] = [
-	"red",
-	"yellow",
-	"green",
-	"blue",
-	"indigo",
-	"purple",
-	"pink",
-	"gray",
-];
+import { GalleryFeedbackTag } from "../../gallery/src/feedback/tag";
 
 const meta: Meta = {
 	title: "Components/Tag",
 	component: Tag,
 	argTypes: {
 		children: Utils.arg("text"),
-		color: Utils.arg(colors),
+		color: Utils.arg(Object.keys(Tag.colors)),
 	},
 };
 
-Utils.page.component(meta, { sticky: true, shots: [] });
+Utils.page.component(meta, { sticky: true, shots: [<GalleryFeedbackTag />] });
 
 export default meta;
 
 interface Props {
-	children?: string;
-	color?: string;
+	children: string;
+	color: string;
 }
 
 export const Primary = (props: Props): JSX.Element => {
+	const color = Object.entries(Tag.colors).filter(
+		(value) => value[0] == props.color
+	);
+	console.log(color);
 	return (
-		<Tag color={(Tag.colors as any)[props.color ? props.color : "gray"]}>
+		<Tag color={(color && color[0] && color[0][1]) ?? Tag.colors.gray}>
```

Comment:
Sorry but the previous one is better :( I mean this:

```ts
color={(Tag.colors as any)[props.color ?? "gray"] as any}
```

Reply:
It seems like "any" has raise some warming which cost ESLint error.

Reply:
ah yes.. you gonna need an "eslint-disable-next-line" like https://github.com/moaijs/moai/blob/ba2b9830e3cb82d0561e4e42cd5cd86e88866a4c/lib/docs/src/button.stories.tsx#L63-L64